nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/defn/ViewConfig.java

```diff
@@ -2681,6 +2681,45 @@ public static boolean isValidNumericFilter(FilterMode mode) {
 		String type() default "text";
 
 	}
+	
+	
+	/**
+	 * <p>InputMask is a text input component.
+	 * 
+	 * <p><b>Expected Field Structure</b>
+	 * 
+	 * <p>InputMask will be rendered when annotating a field nested under one of
+	 * the following components: <ul> <li>{@link Form}</li>
+	 * 
+	 * <p>InputMask should decorate a field having a simple type.
+	 * 
+	 * @since 1.0
+	 */
+	@Retention(RetentionPolicy.RUNTIME)
+	@Target({ ElementType.FIELD })
+	@ViewStyle
+	public @interface InputMask {
+		String alias() default "InputMask";
+		
+		boolean dataEntryField() default true;
+		
+		/**
+		 * <p> maskStyle can be used to define the restricted format user is expected to enter.
+		 * For example maskStyle="99-9999" would let user enter only 6 numerical values in the i/p box.
+		 */
+		String mask() default "";
+		
+		/**
+		 * <p> Please visit PrimeNG's website for details on the below parameters.
+		 */
```

Comment:
Please duplicate this javadoc for each field -- otherwise the other fields (e.g. `charRegex` and `maskPlaceholder` won't show these docs when generated or on hover in the IDE.